df.viewscreen.xml

```diff
@@ -476,12 +476,93 @@
         <int32_t name='map_x'/>
         <int32_t name='map_y'/>
         <pointer name='site' type-name='world_site'/>
-        <stl-vector name='unk_1'/>
+        <stl-vector name='site_artifacts' pointer-type='artifact_record'/>
         <stl-vector name='unk_2'/>
         <int8_t name='unk_44_11a' since='v0.44.11'/>
         <stl-vector name='unk_44_11b' since='v0.44.11'/>
         unsure about order of these two vectors
-        <stl-vector name='missions'/>
+        <stl-vector name='missions'>
```

Comment:
Any idea if this relates to `world.status.missions` (or maybe the thing after)? They don't look like the same layout at all, but it would be nice to know if some of this data comes from there (or somewhere else).

Reply:
Don't know yet. But when I created another mission, size of vectors in `status` didn't change.

Reply:
Weird, the reason it was named `missions` was because it did change size earlier in 0.44. Maybe this struct type should be named `mission` instead? I'll investigate.

Reply:
Missions in `status` are past missions, and `unk_6` there is report text.

Reply:
And `status.anon_2` are some reports having titles like "Spoils Report: The Abbeys of Filling, Summer 210",